component/storage/mongodb/store_test.go

```diff
@@ -283,22 +283,25 @@ func doAllTests(t *testing.T, connString string) {
 		}))
 	require.NoError(t, err)
 
-	commontest.TestAll(t, provider)
-	testGetStoreConfigUnderlyingDatabaseCheck(t, connString)
-	testMultipleProvidersSettingSameStoreConfigurationAtTheSameTime(t, connString)
-	testMultipleProvidersStoringSameDataAtTheSameTime(t, connString)
-	testMultipleProvidersStoringSameBulkDataAtTheSameTime(t, connString)
-	testCloseProviderTwice(t, connString)
-	testQueryWithMultipleTags(t, connString)
-	testQueryWithLessThanGreaterThanOperators(t, connString)
-	testStoreJSONNeedingEscaping(t, connString)
-	testBatchIsNewKeyError(t, connString)
-	testPing(t, connString)
-	testGetAsRawMap(t, connString)
-	testGetBulkAsRawMap(t, connString)
+	if false {
```

Comment:
Is this intended?

Reply:
That's a mistake. Thanks for catching this! I've reverted those lines back.